### DISCUSSION TITLE
Please add Firebase_Database feature for web

### DISCUSSION THREAD

**Discussion by DarshakVasoya** [firebase/flutterfire#5305] [Feature request] [answered]
Good morning Sir / Ma'am,
I request to you, Please add Firebase_Database feature for web.
if you are working on it, can you give me time when you will launched?
I am waiting your response.
Thank you..

**Comment by deepak786** [comment, marked as answer]
@DarshakVasoya PR https://github.com/FirebaseExtended/flutterfire/pull/6952 is already in Progress for web support.
